hugo/content/guides/formatting.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Formatting"
+weight: 300
+---
+
+Langium's formatting API allows to easily create formatters for your language.
+We start building a custom formatter for our language by creating a new class that inherits from `AbstractFormatter`.
+
+```ts
```

Comment:
Could you specify where to put the `CustomModelFormatter` class? Now it looks like it has to be in one file with `CustomModelModule`.

Reply:
I'd propose don't use `CustomModelFormatter` -- you do all explanations using the domain model, so, maybe it's nice to tell about `DomainModelFormatter` from the beggning.

Reply:
Also, looks like the `CustomModelFormatter` and `CustomModelModule` has to be called `CustomFormatter` and `CustomModule`.